smart-contracts/erc20/assembly/exportable/erc20Core.ts

```diff
@@ -0,0 +1,40 @@
+import { Args } from "@massalabs/as-types";
+import { Address } from "@massalabs/massa-as-sdk";
+
+export function allowance(_args: StaticArray<u8>): StaticArray<u8> {
+  const args = new Args(_args);
+  const owner = args.mustNext<Address>("owner");
```

Comment:
mustNext is a key piece of the FT implementation or do you use it just because it's better?

Reply:
No, it a [new method of the Args](https://github.com/massalabs/as/commit/d59ebfa25bc51fa2ca71947fb9d2f2fe3937e87f) object.